operatorcourier/api.py

```diff
@@ -0,0 +1,56 @@
+"""
+operatorcourier.courier_api module
+
+This module implements the api that should be imported when using the operator-courier package
+"""
+
+import os
+from tempfile import TemporaryDirectory
+import yaml
+from operatorcourier.build import BuildCmd
+from operatorcourier.validate import ValidateCmd
+from operatorcourier.push import PushCmd
+
+def build_and_verify(source_dir=None, files=None):
+    """Build and verify constructs an operator bundle from a set of files then verifies it for usefulness and accuracy.
+    It returns the bundle as a string.
+
+    :param source_dir: Path to local directory of yaml files to be read.
+    :param files: Comma delimited string of files to create bundle with.
+    """
+
+    yamls = []
+
+    if source_dir is not None: 
+        for filename in os.listdir(source_dir):
+            if filename.endswith(".yaml"):
+                with open(source_dir + "/" + filename) as f:
+                    yamls.append(f.read())
+    elif files is not None:
+        yamls = files.split(",")
+
+    bundle = BuildCmd().build_bundle(yamls)
```

Comment:
The CLI suggests that only the source_dir or files should be set. If the API doesn't throw an error, we may want to document that the source_dir has priority.

Reply:
why not to combine both inputs ?

Reply:
Not sure I follow. They are very different inputs. one is a single string that is a path, the other is a list of serialized yaml. do you mean they should just be one parameter and we have code that determines if it is a path or not?

Reply:
No, I want two parameters, just append "files" to yamls via `yaml.extend()` when defined

Reply:
@MartinBasti are you planning on using the cli interface or importing the api module? Your team is the only reason we were planning on supporting passing the files themselves rather than a directory parameter, and this method is only written that way to support the cli use case. If you aren't going to use the cli, I will just make this a python list parameter and remove that param from the cli.